### ISSUE TITLE
[Vue warn] on version 1.1.1

### ISSUE THREAD

**Issue by jedihugo** [janniks/vue-notion#44]
### Version
- Tested on vue-notion version: `1.1.1`, using `npm run dev`
  - This error/warn will not show up in ssg mode `npm run generate`
- This error/warn will not show up in version: `1.1.0`, `0.4.0`, even using `npm run dev`

### How to reproduce (basic implementation): 
```Vue
<template>
  <NotionRenderer :blockMap="blockMap" fullPage />
</template>

<script>
export default {
  data: () => ({ blockMap: null }),
  async asyncData({ $notion }) {
    // get Notion blocks from the API via a Notion pageId
    const blockMap = await $notion.getPageBlocks("8c1ab01960b049f6a282dda64a94afc7");
    return { blockMap }
  },
};
</script>

<style>
@import "vue-notion/src/styles.css"; /* optional Notion-like styles */
</style>
```

### Error Logs:
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Missing required prop: "blockMap"

found in

---> <NotionTextRenderer>
       <NotionFigure>
         <NotionBlock>
           <NotionRenderer>
             <NotionColumn>
               <NotionBlock>
                 <NotionRenderer>
                   <NotionBlock>
                     <NotionRenderer>
                       <NotionPage>
                         <NotionBlock>
                           <NotionRenderer>
                             <Pages/articles/Slug.vue> at pages/articles/_slug.vue
                               <Nuxt>
                                 <VMain>
                                   <VApp>
                                     <Layouts/default.vue> at layouts/default.vue
                                       <Root>
```

```
[Vue warn]: Invalid prop: type check failed for prop "blockMap". Expected Object, got Undefined 

found in

---> <NotionDecorator>
       <NotionTextRenderer>
         <NotionFigure>
           <NotionBlock>
             <NotionRenderer>
               <NotionPage>
                 <NotionBlock>
                   <NotionRenderer>
                     <Anonymous>
                       <Nuxt>
                         <VMain>
                           <VApp>
                             <Layouts/default.vue> at layouts/default.vue
                               <Root>
```

**Comment by janniks** [comment]
Nvm, found the bug — fixed in `1.1.3`

**Comment by jedihugo** [comment]
> Nvm, found the bug — fixed in `1.1.3`

Awesome! Thanks! the console log no longer mad in 1.1.3
